docs/sources/tempo/troubleshooting/long-running-traces.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Long-running traces
+description: Troubleshoot search results when using long-running traces
+weight: 479
+aliases:
+  - ../operations/troubleshooting/long-running-traces/
+---
+
+# Long-running traces
+
+An issue arises in Tempo when a user exercises a usage pattern referred to as
+long-running traces. This happens is when Tempo spans receives spans for a
+trace, and then there is a delay, and then Tempo receives additional spans for
+the same trace. If the delay between spans is great enough, the spans end up in
+different blocks, which can lead to inconsistency in a few ways.
+
+1. When using TraceQL search, the duration information only pertains to the
```

Comment:
here is our effort to combine metadata for the same trace. it is far from perfect, but it does attempt to consolidate across multiple traces.

https://github.com/grafana/tempo/blob/main/pkg/traceql/combine.go#L54

Reply:
I think I've captured this in the update.  Instead of mentioning "a single block", I just mention that not all blocks are used to determine the resulting duration on the search, which I think is accurate.